spec/services/remove_section_service_spec.rb

```diff
@@ -69,19 +69,27 @@
       }.to raise_error(described_class::SectionNotFoundError, document_id)
     end
 
-    it "does not mark the manual as a draft" do
-      begin; service.call; rescue; end
-      expect(manual).not_to have_received(:draft)
-    end
+    context "when SectionNotFoundError is ignored" do
```

Comment:
I wonder whether this would be better named as "when SectionNotFoundError is raised"?